sdk/formrecognizer/Azure.AI.FormRecognizer/src/FormField.cs

```diff
@@ -30,6 +35,25 @@ internal FormField(KeyValuePair_internal field, ReadResult_internal readResult)
             //}
         }
 
+        internal FormField(string name, FieldValue_internal fieldValue, IReadOnlyList<ReadResult_internal> readResults)
+        {
+            Confidence = fieldValue.Confidence ?? 1.0f;
```

Comment:
We don't have to do it now, but if there are more places where we need to set the default confidence value, we could do something like `Constants.DefaultConfidenceValue` so we have one value across all library